src/transactions/transferFeedUtils.ts

```diff
@@ -400,3 +426,53 @@ export function isTransferTransaction(
 ): tx is TransferItemFragment {
   return (tx as TransferItemFragment).address !== undefined
 }
+
+// Note: This hook is tested from src/transactions/feed/TransferFeedItem.test.ts
+function useFiatConnectTransferDisplayInfo({ amount, transactionHash }: TokenTransfer) {
+  const { t } = useTranslation()
+  const tokenInfo = useTokenInfo(amount.tokenAddress)
+  const fcTransferDetails = useSelector(getCachedFiatConnectTransferSelector(transactionHash))
+  const cachedFiatAccountUses = useSelector(cachedFiatAccountUsesSelector)
+  const account = useMemo(
+    () =>
+      fcTransferDetails
```

Comment:
I think this should work?

```suggestion
      cachedFiatAccountUses.find(
        ({ fiatAccountId }) => fiatAccountId === fcTransferDetails?.fiatAccountId
      ),
```

Reply:
Yeah I had it like that previously, but it [was suggested](https://github.com/valora-inc/wallet/pull/3407#discussion_r1092494511) that I terminate early so that a `find` is not done unnecessarily.